cross-platform/react-app/src/frontend/App.tsx

```diff
@@ -256,4 +260,27 @@ function App() {
   }
 }
 
-export default App;
+export function CameraSampleApp() {
```

Comment:
I'm not thrilled with this copying so much from App().

Reply:
I refactored things more by adding AppProps, and moved CameraSampleApp into the CameraSample folder. 

The AppProps has an optional function that returns the components the CameraSample is going to add.

The useAppState is no longer used outside of App.tsx so it could be moved back into the App function, for now I've removed the export.